.github/workflows/forecr-dsb-ornx-orin-nano-8gb.yml

```diff
@@ -0,0 +1,44 @@
+name: Forecr DSB ORNX LAN Nano 8GB
+
+on:
+  pull_request:
+    branches:
+      - main
+      - master
+      # ESR branches glob pattern
+      - "[0-9]+.[0-9]+.x"
+  pull_request_target:
+    branches:
+      - main
+      - master
+      # ESR branches glob pattern
+      - "[0-9]+.[0-9]+.x"
+
+permissions:
+  id-token: write # This is required for requesting the JWT #https://docs.github.com/en/actions/deployment/security-hardening-your-deployments/configuring-openid-connect-in-amazon-web-services#requesting-the-access-token
+  actions: read # We are fetching workflow run results of a merge commit when workflow is triggered by new tag, to see if tests pass
+  pull-requests: write # Read is required to fetch the PR that merged, in order to get the test results. Write is required to create PR comments for workflow approvals.
+  packages: read
+  contents: read
+
+jobs:
+  yocto:
+    name: Yocto
+    # TODO: Add a renovate versioning strategy to support tags like v6.5.19+rev3 and avoid pinning to master
+    uses: balena-os/balena-jetson-orin/.github/workflows/forecr-dsb-ornx-orin-nano-8gb.yml@ryan/workflow-call
+    # Prevent duplicate workflow executions for pull_request (PR) and pull_request_target (PRT) events.
+    # Both PR and PRT will be triggered for the same pull request, whether it is internal or from a fork.
+    # This condition will prevent the workflow from running twice for the same pull request while
+    # still allowing it to run for all other event types.
+    if:
+      (github.event.pull_request.head.repo.full_name == github.repository) ==
+      (github.event_name == 'pull_request')
+    secrets: inherit
+    with:
+      # Needed for testing - defaults to production
+      deploy-environment: balena-staging.com
```

Comment:
There's been some discussion that we may reduce the architectures of the supervisor we are publishing to staging, either temporarily or permanently, so in order for these builds to pass we should change all of these deploy-environments to `balena-cloud.com` (or just remove the override)

Nothing deploys in this workflow anyway, this will just ensure that everything still builds if we skip publishing SV armv7 to staging